draft-fossati-tls-attestation.md

```diff
@@ -154,74 +154,108 @@ Several formats for encoding evidence are available, such as:
 - the Android Key Attestation, and
 - Apple Key Attestation. 
 
-Likewise, there are different encodings available for attestation results.
-One such encoding, AR4SI {{?I-D.ietf-rats-ar4si}} is being standardized by the RATS 
+Likewise, there are different encodings available for attestation results. One
+such encoding, AR4SI {{?I-D.ietf-rats-ar4si}} is being standardized by the RATS
 working group.
 
-This version of the specification defines how to support the background check model
-in  the TLS handshake, such that the details about the attestation technology are
-agnostic to the TLS handshake itself. Later versions of the specification will 
-support the passport model as well. 
+This specification defines how to support the background check model in the TLS
+handshake, such that the details about the attestation technology are agnostic
+to the TLS handshake itself. Similarly, support for the passport model is
+defined such that details about the attestation results encodings and trust
+relationships are agnostic to the TLS handshake.
+
+To give the peer information that the handshake signing key is properly secured,
+the associated attestation result has to be appraised by the peer. This must be
+the case when either of the two remote attestation topologies is used. Hence,
+attestation evidence about the security state of the signing key is needed,
+which is typically associated with evidence about the overall platform state.
```

Comment:
Hence the attestation evidence (to be appraised by an attestation verifier )
must contain the security state of the signing key and also the security of the
overall platform state.

Reply:
attestation verifier is not used otherwise in the draft, so probably skip "attestation". 

Also, slightly shorter and concise version: 

Hence, the attestation evidence (to be appraised by the verifier)
must contain the security state of the signing key as well as the overall platform.